lib/mix/tasks/transport/validate_data.ex

```diff
@@ -1,35 +1,36 @@
 defmodule Mix.Tasks.Transport.ValidateData do
   @moduledoc """
-  Queues all ok data to validation
+  Queues all ok data for validation.
   """
 
   use Mix.Task
-  alias Transport.ReusableData.Dataset
+  alias Transport.ReusableData
   alias Transport.DataValidation
   require Logger
 
-  @pool DBConnection.Poolboy
+  @concurrency 5
+  @timeout :infinity
```

Comment:
The biggest file we can have, is the GTFS of ÎdF. It’s currently 60Mb, but might grow up to 100Mb at certain moments of the year.
The validation takes 15seconds, so I guess that 1 minute would be a reasonable timeout.

Concerning the concurrency, right now the validator is able to process everything. So a 5 parallel should be fairly conservative

Reply:
OK